### PR TITLE
Fix AFL License header template

### PR DESCRIPTION
**Pull request by matks**

### CODE REVIEW

**Comment by matks** [comment]
Fixes https://github.com/PrestaShopCorp/header-stamp/issues/31

File: assets/afl.txt [hunk 1]
```diff
@@ -4,15 +4,15 @@
  *
  * NOTICE OF LICENSE
  *
- * This source file is subject to the Academic Free License 3.0 (AFL-3.0)
+ * This source file is subject to the Academic Free License (AFL 3.0)
```
Comment (PierreRambaud) [inline assets/afl.txt:7]:
```suggestion
 * This source file is subject to the Academic Free License version 3.0
```

https://opensource.org/licenses/AFL-3.0
https://choosealicense.com/licenses/afl-3.0/

File: assets/afl.txt [hunk 2]
```diff
@@ -4,15 +4,15 @@
...
  * https://opensource.org/licenses/AFL-3.0
  * If you did not receive a copy of the license and are unable to
  * obtain it through the world-wide-web, please send an email
  * to license@prestashop.com so we can send you a copy immediately.
  *
- * @author    PrestaShop SA <contact@prestashop.com>
+ * @author    PrestaShop SA and Contributors <contact@prestashop.com>
  * @copyright Since 2007 PrestaShop SA and Contributors
- * @license   https://opensource.org/licenses/AFL-3.0 Academic Free License 3.0 (AFL-3.0)
+ * @license   https://opensource.org/licenses/AFL-3.0  Academic Free License (AFL 3.0)
```
Comment (PierreRambaud) [inline assets/afl.txt:17]:
```suggestion
 * @license   https://opensource.org/licenses/AFL-3.0 Academic Free License version 3.0
```

**Comment by matks** [comment]
If I get it right, you say https://devdocs.prestashop.com/1.7/development/coding-standards/#module-files is wrong too ?

**Comment by PierreRambaud** [comment]
> If I get it right, you say https://devdocs.prestashop.com/1.7/development/coding-standards/#module-files is wrong too ?

Yep :(

**Comment by Quetzacoalt91** [comment]
Hi @matks & @PierreRambaud,

How can we go forward on this matter? Shall we update the afl license for a public module then update the documentation accordingly?

**Comment by PierreRambaud** [comment]
> Hi @matks & @PierreRambaud,
> 
> How can we go forward on this matter? Shall we update the afl license for a public module then update the documentation accordingly?

Yes, we should update both

**Comment by Quetzacoalt91** [comment]
I guess we have to update the folder `tests/integration/expected` according to the new template.

Doo you have time to handle that @matks by any chance?

**Review by PierreRambaud — APPROVED** [review]

**Comment by Quetzacoalt91** [comment]
Thank you  @matks

**Pull request merged**